spec/services/FetchMoviesSpec.php

```diff
@@ -33,19 +32,16 @@
             expect($movieData->release_date)
             ->toBeA('string');
             expect($movieData->poster_image)
-            ->toBeA('null');
```

Comment:
looks like here you are testing the movie data which is technically outside the scope of the testing the Fetcher

Reply:
Correct, I was initially testing to make sure it was getting the correct data in each property of the movie class instance. Instead I should just test the return of an instance of a movie class?